packages/node_modules/@ciscospark/widget-recents/src/components/RecentsHeader.js

```diff
@@ -2,31 +2,34 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames';
 
-import {Button, Icon, SearchInput} from '@collab-ui/react';
+import {Button, Icon, Popover, SearchInput} from '@collab-ui/react';
 
 import PresenceAvatar from '@ciscospark/react-container-presence-avatar';
 
-
 import styles from './RecentsHeader.css';
+import PopoverContent from './PopoverContent.js';
```

Comment:
You don't need the `.js` here. Also, can you move it above the styles import? I don't think there's an actual "rule" but the style import is the last import on (hopefully all) of our components.